.travis.yml

```diff
@@ -1,19 +1,26 @@
-language: python
+language: shell
 sudo: required
 env:
   global:
     - LD_PRELOAD=/lib/x86_64-linux-gnu/libSegFault.so
     - SEGFAULT_SIGNALS=all
     - SALT_STATE=name-computer
     - SALT_GRAIN_KEY='name-computer:computername'
-  matrix:
-    - OS_VERSION=6 SALT_GRAIN_VALUE=False
-    - OS_VERSION=6 SALT_GRAIN_VALUE=itworx
-    - OS_VERSION=7 SALT_GRAIN_VALUE=False
-    - OS_VERSION=7 SALT_GRAIN_VALUE=itworx
 
-services:
-  - docker
+matrix:
+  include:
+    - env:
+        - OS_VERSION=7
+        - SALT_GRAIN_VALUE=False
+        - SALT_REPO_URL=https://repo.saltproject.io/py3/redhat/7/x86_64/3004.repo
+      services:
+        - docker
+    - env:
+        - OS_VERSION=7
+        - SALT_GRAIN_VALUE=itworx
+        - SALT_REPO_URL=https://repo.saltproject.io/py3/redhat/7/x86_64/3004.repo
+      services:
+        - docker
```

Comment:
Presumably, when we moved to 3005.1+ (for EL8 compatibility), we'll need to add testing-supporting values for EL8?